### PR TITLE
Frexp ldexp params

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: test/function/simd/toint.more.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  using p_t = bs::pack<T>;
+  using r_t = decltype(toint(p_t()));
+  STF_TYPE_IS(r_t, r_t);
+
+// for(int64_t i=4294967296; i <8589934592*1000 ; i+= 100001)
+//  {
+//    int64_t j1 =  i&3ul;
+//    r_t z = bs::bitwise_and(toint(p_t(i)), r_t(3));
+//    if (z[0] !=  j1) {
+//      std::cout << "failed at " << i << std::endl;
```
Comment (jfalcou) [inline test/function/simd/toint.more.cpp]: remove dont comment

File: test/function/simd/toint.more.cpp [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+//    }
+
+//  }
+
+
+  for(int64_t i=0; i < 10/*4.2950e+09*/ ; i++)
+ {
+//    int64_t j1 =  i&3ul;
+    r_t z = bs::bitwise_and(toint(p_t(i)), r_t(3));
+    int64_t e = ffs(tofloat(i));
```
Comment (jfalcou) [inline test/function/simd/toint.more.cpp]: we dont actuly test there, no STF_* are used so is this file required at all ?

File: include/boost/simd/arch/common/simd/function/ldexp.hpp [hunk 3]
```diff
@@ -236,35 +222,34 @@ namespace boost { namespace simd { namespace ext
...
-    {
-      return ldexp(a0, toint(a1));
-    }
-  };
+//   BOOST_DISPATCH_OVERLOAD ( ldexp_
+//                           , (typename A0, typename A1, typename X)
+//                           , bd::cpu_
+//                           , bs::pack_<bd::single_<A0>, X>
+//                           , bd::scalar_<bd::single_<A1>>
+//                           )
```
Comment (jfalcou) [inline include/boost/simd/arch/common/simd/function/ldexp.hpp]: remove dont comment

File: include/boost/simd/arch/common/simd/function/ldexp.hpp [hunk 4]
```diff
@@ -236,35 +222,34 @@ namespace boost { namespace simd { namespace ext
...
-      return a0*expocvt(a1);
-    }
-  };
+//   BOOST_DISPATCH_OVERLOAD ( ldexp_
+//                           , (typename A0, typename A1, typename X)
+//                           , bd::cpu_
+//                           , bs::pack_<bd::double_<A0>, X>
+//                           , bd::scalar_<bd::double_<A1>>
+//                           )
+//   {
```
Comment (jfalcou) [inline include/boost/simd/arch/common/simd/function/ldexp.hpp]: remove dont comment

File: test/function/simd/div/div.round.cpp [hunk 5]
```diff
@@ -20,6 +20,7 @@ void test(Env& $)
   using p_t = bs::pack<T, N>;
 
   T a1[N], a2[N], b[N];
+  a1[0] = a2[0] = T(0); // suppress a warning !?
```
Comment (jfalcou) [inline test/function/simd/div/div.round.cpp:23]: which warning did you get there ?

**Review by jfalcou — CHANGES_REQUESTED** [review]
did you cross-checked bench performacnes for the affectd functions ?

File: test/function/simd/toint.more.cpp (hunk 1 above)
Comment (jtlap) [reply test/function/simd/toint.more.cpp]: yes

File: test/function/simd/toint.more.cpp (hunk 2 above)
Comment (jtlap) [reply test/function/simd/toint.more.cpp]: this is a spurious file

File: include/boost/simd/arch/common/simd/function/ldexp.hpp (hunk 3 above)
Comment (jtlap) [reply include/boost/simd/arch/common/simd/function/ldexp.hpp]: done

File: include/boost/simd/arch/common/simd/function/ldexp.hpp (hunk 4 above)
Comment (jtlap) [reply include/boost/simd/arch/common/simd/function/ldexp.hpp]: done

File: test/function/simd/div/div.round.cpp (hunk 5 above)
Comment (jtlap) [reply test/function/simd/div/div.round.cpp:23]: a1 and a2 used uninitialized ! which of course is not true !?

**Review by jfalcou — APPROVED** [review]
